client/src/containers/backend/Content/Pages/Detail.js

```diff
@@ -257,10 +260,19 @@ class PageDetailContainer extends PureComponent {
   render() {
     const page = this.page(this.props);
     const isNew = this.isNew(this.props);
+    const action = isNew ? "create" : "update";
 
-    if (isNew) return this.renderNew();
-    if (page) return this.renderExisting(page);
-    return null;
+    return (
+      <HigherOrder.Authorize
+        entity="feature"
+        failureFatalError={{
```

Comment:
Same issue as above with features. Need to authorize the instance in the case of update.